awx/main/access.py

```diff
@@ -2810,22 +2815,22 @@ def can_approve_or_deny(self, obj):
 
 class WorkflowApprovalTemplateAccess(BaseAccess):
     '''
```

Comment:
Same comment here regarding when they can _see_ these:

A user can see approval templates if have read access to the associated WorkflowJobTemplate.